static/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -224,8 +224,15 @@ export class IntegrationListDirectory extends AsyncComponent<
     return integrations?.find(i => i.provider.key === integration.key) ? 2 : 0;
   }
 
-  getPopularityWeight = (integration: AppOrProviderOrPlugin) =>
-    POPULARITY_WEIGHT[integration.slug] ?? 1;
+  getPopularityWeight = (integration: AppOrProviderOrPlugin) => {
+    for (const sentryapp of this.state.publishedApps || []) {
+      if (sentryapp === integration) {
+        return integration.popularity ?? 1;
+      }
+    }
```

Comment:
[publishedApps.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) might be a bit cleaner!